tests/ui/unwrap_or.stderr

```diff
@@ -6,5 +6,13 @@ error: use of `unwrap_or` followed by a function call
   |
   = note: `-D or-fun-call` implied by `-D warnings`
 
-error: aborting due to previous error
+error: use of `unwrap_or` followed by a function call
+ --> $DIR/unwrap_or.rs:8:46
+  |
+8 |       let s = Some(String::from("test string"))
+  |  ______________________________________________^
+9 | |         .unwrap_or("Fail".to_string())
+  | |______________________________________^ help: try this: `.unwrap_or_else(|| "Fail".to_string())`
```

Comment:
I think this output needs to be double checked. 

Looking at it now.

Reply:
If we apply this with rustfix then we lose the newline - but the old code does that too so this isn't a regression.

Reply:
hmm... I think that was the entire point of the opened issue...

We could create a multi_sugg, one for replacing `unwrap_or` with `unwrap_or_else` and one for replacing `"Fail".to_string()` with `|| "Fail".to_string()`

Reply:
Yes i think you are right. I got so sidetracked with moving the '^'s to only point at the part of the code that needs changing.

I'll have a play with it and see if I can come up with something.

Reply:
Pushed a new version. 
Thanks for taking the time to nudge me through this. 
Can we squash all these commits when we merge? I'm conscious that I have gone back and forward quite a bit.